.circleci/config.yml

```diff
@@ -22,10 +22,10 @@ aliases:
     - run:
         name: "Import signing key"
         command: |
+          gpg --keyserver keyserver.ubuntu.com \
+            --recv-keys 0x713F9F29598CFFF3 && \
```

Comment:
This is the [Azavea public GPG key](https://keyserver.ubuntu.com/pks/lookup?search=azavea&fingerprint=on&op=index).